tests/ts_cluster_comprehensive.erl

```diff
@@ -56,7 +56,7 @@ run_tests(PvalP1, PvalP2) ->
                   ?PKEY_P1, ?PKEY_P2, ?PKEY_P3,
                   ?PKEY_P1, ?PKEY_P2, ?PKEY_P3,
                   ?PKEY_P1, ?PKEY_P2, ?PKEY_P3]),
-    ts_util:create_and_activate_bucket_type(Cluster, lists:flatten(TableDef), ?BUCKET),
+    ?assertEqual({ok, {[], []}}, riakc_ts:query(rt:pbc(hd(Cluster)), TableDef)),
```

Comment:
Seems like you could call `ts_util:cluster_and_connect` above to get a cluster and a connection, then use the connection here instead of calling `rt:pbc` directly.  It also verifies that the connection is in place.